.storybook/mdx/highlight/highlight.css

```diff
@@ -1,7 +1,7 @@
 .highlight {
-    background-color: var(--o-ui-bg-alias-accent-light);
+    background-color: var(--hop-primary-surface-weak);
     padding: var(--o-ui-sp-3);
```

Comment:
This var is in the sharegate theme, it should be replaced.

Reply:
I don't follow you.

Reply:
the var for padding should be replaced: var(--o-ui-sp-3) with something from hopper